src/entity_extraction/training/spacy_ner/spacy_transformer_train.cfg

```diff
@@ -132,8 +127,8 @@ initial_rate = 0.00005
 
 [training.score_weights]
 ents_f = 0.0
-ents_p = 0.3
-ents_r = 0.7
+ents_p = 0.5
+ents_r = 0.5
```

Comment:
No, the spacy v3 model was trained with the older numbers. Since we are open-sourcing the solution, I left it equally weighted. This sample file will be used to create a config file on the fly and hasn't been used to train any of our models.